R/include-hba1c.R

```diff
@@ -15,13 +15,13 @@
 #' }
 include_hba1c <- function(data) {
   verify_required_variables(data, "lab_forsker")
-  hba1c_criteria <- get_algorithm_logic("hba1c") |>
+  criteria <- get_algorithm_logic("hba1c") |>
     # To convert the string into an R expression.
     rlang::parse_expr()
   data |>
     column_names_to_lower() |>
     # Use !! to inject the expression into filter.
-    dplyr::filter(!!hba1c_criteria) |>
+    dplyr::filter(!!criteria) |>
```

Comment:
Regarding line 36-37: We should actually not slice the data to the first two observations yet. This has to wait until after the events have been through the censoring function(s). E.g. if we do it now, we risk that one or both of the first two elevated HbA1c tests were due to gestational diabetes and need to be excluded.
The same thing goes for GLD drugs (which need to go through three censoring functions, while HbA1c only needs to go through one: exclude_pregnancy()), but include_gld_purchases() doesn't slice the data, so that function is fine 👍

Reply:
I have made an issue on this, so it can be fixed in another PR. Will resolve this conversation 👍